tests/test_packages.py

```diff
@@ -67,7 +67,7 @@ def test_simple_setuptools(tmp_path, env, nvcc_version):
     }
 
     package_dir = tmp_path / "pkg"
-    os.makedirs(package_dir)
+    os.makedirs(package_dir / "simple_setuptools")
```

Comment:
Why did this directory (and the one below) change?

Reply:
This is where RBB puts the `GIT_COMMIT` file, and the directory has to exist, otherwise it throws a file not found error.